src/js/types/types.ts

```diff
@@ -28,3 +31,11 @@ export type TableItemChangeDetailType = {
 export type TableItemDeleteDetailType = {
   item: ItemType;
 };
+
+export type userDataType = {
```

Comment:
```suggestion
export type UserDataType = {
```
나머지는 파스칼 케이스를 사용해주셨는데 이 타입은 놓치신거 같아요

Reply:
👀 수정했습니다.